docs/usage/docker.md

```diff
@@ -117,6 +117,27 @@ If you wish to enable major versions then add the preset `docker:enableMajor` to
 Renovate has some Docker-specific intelligence when it comes to versions.
 For example:
 
+### Ubuntu codenames
+
+Renovate understands [Ubuntu release codename](https://wiki.ubuntu.com/Releases) tags and will offer upgrading them to the latest LTS release available (e.g. from `ubuntu:xenial` to `ubuntu:focal`).
```

Comment:
```suggestion
Renovate understands [Ubuntu release code names](https://wiki.ubuntu.com/Releases) and will offer upgrades to the latest LTS release (e.g. from `ubuntu:xenial` to `ubuntu:focal`).
```

- We can drop "tags" and just use "code names".
- Replace `upgrading them` with `upgrades`.
- People won't expect Renovate to update things that "are not available". So we can drop the "available" part. 😉